core/bdo.ttl

```diff
@@ -2718,13 +2751,24 @@ bdo:hasFather
 	rdfs:subPropertyOf bdo:hasParent ;
 	rdfs:label "has father"@en ;
 	adm:userTooltip "Indicates the Father of the Person"@en ;
+	rdfs:range [
+		a owl:Restriction ;
+		owl:onProperty bdo:personGender ;
+		owl:hasValue bdr:GenderMale ;
+	] ;
+	skos:definition "Indicates the Father of the Person"@en ;
```

Comment:
it's a tiny detail and I know French uses less upper case letter than English... but why an upper case F here? it makes it look very divine!

Reply:
I was following the pattern used throughout BDO. If you look at the admin tooltips, which I simply copied to `skos:definition`s, you'll see that all the relationship terms are capitalized. I would also prefer to uncapitalize them. If you agree with that, I'll go through and make the changes to both annotations.

Reply:
ah I see yes, I think it makes more sense to use the normal (lower) case here... I think the original intent was to highlight that the words were referring to classes (sort of)... but as we encode things from the real world, we can use the case that's appropriate for these. Thanks!